text/_util.ts

```diff
@@ -1,9 +1,25 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+function splitByCase(input: string): string[] {
+  const acronym = /\p{Lu}{2,}/gu; // e.g. ID, URL
+  const capitalizedWord = /\p{Lu}\p{Ll}+/gu; // e.g. Apple
+  const lowercasedWord = /\P{Lu}+/gu; // e.g. apple, this will also match a sequence of unicode letters in languages without a concept of upper/lower case
+  const digits = /\p{N}+/gu; // e.g. 123
```

Comment:
Nit: can you please move these out of the function? Otherwise, they're declared every time `splitByCase()` is called. Also, nice comments.